solidity/deploy/42_register_solvbtc.ts

```diff
@@ -0,0 +1,37 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
Wouldn't it be easier to accept the initial supported tokens in the `initialize` function, just [like in the Mezo Portal](https://github.com/thesis/mezo-portal/blob/f09b08ea26274a8c3694aabff82821f35f38dc16/solidity/contracts/Portal.sol#L478)? We could event hardcode them in the contract. We would reduce the deployment to just one script without the need to involve multisig to accept the ownership. 

`0x790Dda4c56b3c45d0e4514eDbAaBa30D7129c857` (the [`governance`](https://github.com/thesis/acre/blob/cb5f67c801b4263c3463b3b68352f11f47b8ae48/solidity/hardhat.config.ts#L101)) kept executing transactions in the past so we know it works and do not have to test it by accepting the ownership.

Reply:
4b7d865fbc5dfc254f51d91200cbc77f867d7998